Potential Cra-Template Issues

I am continuously getting the following below error when running 

```
npx create-react-app drain-safe --template @gnosis.pm/cra-template-safe-app
```

I believe it has to do with the version here https://github.com/safe-global/safe-apps-sdk/blob/master/packages/cra-template-safe-app/template.json#L15.

see https://github.com/testing-library/react-testing-library/issues/1121#issuecomment-1242335979 for example.

Any idea what might be going on here? Thanks.

```
npm ERR!   react@"^17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.0.0" from @testing-library/react@13.4.0
npm ERR! node_modules/@testing-library/react
npm ERR!   dev @testing-library/react@"^13.4.0" from the root project
```

Curious what I may be missing on my end, thanks!

Have attempted to move down to react@17.0.2 and react-dom@17.0.2 to no avail.

----------------------------------------------------

Update: I have gotten the command to not error out. For anyone looking to get this up and running for the time being:

the following fails:
```npx create-react-app drain-safe --template @gnosis.pm/cra-template-safe-app@4.2.0```

and the following works:
1. ```npx create-react-app drain-safe --template @gnosis.pm/cra-template-safe-app@4.1.1```. 
Again, I believe there are some broken fonts in this version but the command does not error out.

2. Remove related font code here https://github.com/safe-global/safe-apps-sdk/pull/388/files#diff-68c12a430f865c61eee3d09fe68f486c048af196cf7a2b347f0c5105c94f01c8L1-L2.

3. ```npm start``` (or yarn)

--- a/.changeset/friendly-students-tickle.md
+++ b/.changeset/friendly-students-tickle.md
@@ -1,0 +1,5 @@
+---
+'@gnosis.pm/cra-template-safe-app': patch
+---
+
+Downgrade @testing-library/react to the version that supports react 17

--- a/packages/cra-template-safe-app/template.json
+++ b/packages/cra-template-safe-app/template.json
@@ -12,7 +12,7 @@
     },
     "devDependencies": {
       "@testing-library/jest-dom": "^5.16.5",
-      "@testing-library/react": "^13.4.0",
+      "@testing-library/react": "^12.1.5",
       "@testing-library/user-event": "^14.4.3",
       "@types/jest": "^29.2.0",
       "@types/node": "^18.11.7",

